lib/rust/ensogl/component/grid-view/src/lib.rs

```diff
@@ -258,7 +258,7 @@ impl<Entry: display::Object, EntryParams> Model<Entry, EntryParams> {
         for ((row, col), visible_entry) in &*self.visible_entries.borrow() {
             let size = properties.entries_size;
             let widths = &self.column_widths;
-            entry::visible::set_position(visible_entry, *row, *col, size, widths);
+            entry::visible::set_position(&visible_entry.entry, *row, *col, size, widths);
```

Comment:
Why is this needed?

Reply:
We need to access the `Entry` frp, so we need to know that this is an `Entry`, not just a `display::Object`. `visible_entry` is not an `Entry`, but does deref to a display object.